File: MakeGood.Server/Models/Dtos/NewEnquiryDto.cs
namespace MakeGood.Server.Models.Dtos;

public class NewEnquiryDto
{
    public string Name { get; set; }

    public string Email { get; set; }

    public string Phone { get; set; }

    public string Message { get; set; }

    public string Captcha { get; set; }
}

File: makegood.client/src/app/models/apis/NewEnquiryDto.ts
import { NewEnquiryFormGroup } from '../forms/NewEnquiryFormGroup';

export class NewEnquiryDto {
  name: string;
  email: string;
  phone: string;
  message: string;
  captcha: string;

  update(formGroup: NewEnquiryFormGroup) {
    this.name = formGroup.name.value;
    this.email = formGroup.email.value;
    this.phone = formGroup.phone.value;
    this.message = formGroup.message.value;
    this.captcha = formGroup.captcha.value;
  }
}
